x/tier/keeper/lockup.go

```
@@ -340,3 +361,32 @@ func (k Keeper) lockupStore(ctx context.Context, unlocking bool) prefix.Store {
 	storePrefix := types.KeyPrefix(unlocking)
 	return prefix.NewStore(storeAdapter, storePrefix)
 }
+
+// GetTotalLockupsAmount retrieves the total lockup amount from the store.
+func (k Keeper) GetTotalLockupsAmount(ctx context.Context) (total math.Int) {
+	store := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+	bz := store.Get(types.TotalLockupsKey)
+	if bz == nil {
+		return math.ZeroInt()
+	}
+
+	err := total.Unmarshal(bz)
+	if err != nil {
+		return math.ZeroInt()
```

question: do we want to ignore this error? It's unlikely to happen, but if it does it means some code invariant is broken. Would be nice to at least log it